Sample/SampleDelegate/SampleDelegate/AppDelegate.swift

```diff
@@ -100,62 +100,62 @@ extension AppDelegate: RemoteNotificationCapable {
 
 extension AppDelegate: UserNotificationCapable {
     func requestedUserNotificationSettings() -> UIUserNotificationSettings {
-        return UIUserNotificationSettings(forTypes: [.Badge, .Sound, .Alert], categories: nil)
+        return UIUserNotificationSettings(types: [.badge, .sound, .alert], categories: nil)
     }
     
-    func didReceiveUserNotificationPermissions(userNotificationPermissionsGranted: UserNotificationPermissionsGranted) {
-        switch userNotificationPermissionsGranted {
-        case .None:
+    func didReceive(userNotificationPermissions: UserNotificationPermissionsGranted) {
+        switch userNotificationPermissions {
+        case .none:
             // The customer has denied user (e.g. local and loud remote) notification permissions.
             break
-        case .Requested:
+        case .requested:
             // The customer has given us permission for all requested user notifications.
             break
-        case let .Some(grantedPermissions):
-            print("Customer refused to give us permissions for \(requestedUserNotificationSettings().types.exclusiveOr(grantedPermissions))")
+        case let .some(grantedPermissions):
+            print("Customer refused to give us permissions for \(requestedUserNotificationSettings().types.symmetricDifference(grantedPermissions))")
```

Comment:
Not sure if there was comment on it, but should we be using NSLog? "print" just goes to stdout and don't believe it ends up in the device log or syslogs unlike NSLog does.